### PR TITLE
Resolve #112

### LINKED ISSUE #112: [FEATURE] - Light YouTube Embedded
**I have checked all the prerequisites below and I don't see this is available**

- [x] Read the README.md
- [x] Have the extended version of Hugo installed
- [x] Used the exampleSite's config.toml as a reference

Lite YouTube Embedded
https://github.com/paulirish/lite-youtube-embed

I added the JS and the CSS to my head.html partial, and it loads the YouTube embed *much* faster.

I only implemented a very minimal part of this script.

I created a shortcode "liteyoutube.html" with this code in it:
<lite-youtube videoid="{{ .Get `videoid` }}"></lite-youtube>

The markdown script just had this in it:
{{< liteyoutube videoid="your_vid_id" >}}

Just a thought.

### PR DESCRIPTION
**Pull request by rwstorer**
This PR...
Adds liteyoutube by paulirish found [here](https://github.com/paulirish/lite-youtube-embed)

## Changes / fixes

- Resolves #112 
- Adds shortcode liteyoutube.html
- Adds JS static/js/lite-yt-embed.js
- Adds CSS static/css/lite-yt-embed.css
- Modifies Doc exampleSite/content/docs/compose/shortcodes.md

## Screenshots (if applicable)

(prefer animated gif)

## Checklist

_Ensure you have checked off the following before submitting your PR._

- [x] tested locally with the [latest release of Hugo](https://github.com/gohugoio/hugo/releases). This requirement is [a standard](https://github.com/gohugoio/hugoThemes#theme-maintenance)
- [x] added new dependencies
- [x] updated the [docs]() ⚠️

### CODE REVIEW

**Comment by rwstorer** [comment]
Sorry, @onweru. I do not know how to fix this failed check. The code works on my machine. All my check-ins are "verified."

**Comment by onweru** [comment]
> Sorry, @onweru. I do not know how to fix this failed check. The code works on my machine. All my check-ins are "verified."

No problem, I'll review this asap i.e from Monday. Weekends can are kinda swarmed on my end 😃

File: exampleSite/content/docs/compose/shortcodes.md
Comment (rwstorer) [inline exampleSite/content/docs/compose/shortcodes.md:1]: I just documented the liteyoutube shortcode with some examples.

File: layouts/shortcodes/liteyoutube.html
Comment (rwstorer) [inline layouts/shortcodes/liteyoutube.html:1]: This is the liteyoutube shortcode file. I borrowed the code from [https://github.com/justinribeiro/lite-youtube](https://github.com/justinribeiro/lite-youtube). Then, I modified it for both named and positional parameters. I forgot to put the credit for the original code in the source. We can add it.

Comment (rwstorer) [reply layouts/shortcodes/liteyoutube.html:1]: Sorry. No. I did not get it from the above URL. I got it from here [https://github.com/paulirish/lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed)

File: static/css/lite-yt-embed.css
Comment (rwstorer) [inline static/css/lite-yt-embed.css:1]: This is the CSS file for the liteyoutube component found here [https://github.com/paulirish/lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed). I forgot to add the credit to the CSS file. We should add it.

File: static/js/lite-yt-embed.js
Comment (rwstorer) [inline static/js/lite-yt-embed.js:1]: This is the JavaScript file for the liteyoutube component. Again, you can find it here [https://github.com/paulirish/lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed). Again, I forgot to put the credit line in this source. We should add it.

**Comment by rwstorer** [comment]
@onweru, I figured out that I needed to point the "exampleSite" to my repo during the test import for it to find the new shortcode. Naturally, you'll want to update that after the merge (or reject that change).

Anyway, the PR passes all checks now.

**Pull request merged**